orchagent/portsorch.cpp

```diff
@@ -2937,12 +2968,37 @@ void PortsOrch::doLagTask(Consumer &consumer)
                         continue;
                     }
                 }
+                else if (fvField(i) == "lag_id")
+                {
+                    lag_id = (uint32_t)stoul(fvValue(i));
+                }
+                else if (fvField(i) == "switch_id")
+                {
+                    switch_id = stoi(fvValue(i));
+                }
+            }
+
+            if (table_name == CHASSIS_APP_LAG_TABLE_NAME)
+            {
+                if (switch_id == gVoqMySwitchId)
+                {
+                    //Already created, syncd local lag from CHASSIS_APP_DB. Skip
+                    it = consumer.m_toSync.erase(it);
+                    continue;
```

Comment:
Is it always safe to simply skip here? Is it possible this is to retry a lag that was failing to add?

Reply:
Yes. We should always skip this. This lag is the Portchannel locally created already and synced to CHASSIS_APP_DB. Since this api, which is used for both local lags and remote lags, this is invoked for entries added to CHASSIS_APP_DB also. Here we identify the local entry by the switch id. If it is local entry, since this local is already programmed, we should always skip this and erase it. So there will not be re-try at all.